README.md

```
@@ -0,0 +1,37 @@
+# Contenido
+
+- [Instalación](#instalación)
+- [Uso](#uso)
+- [Contribuciones](#contribuciones)
+- [Licencia](#licencia)
+
+## Instalación
+
+Para instalar la librería, simplemente ejecute el siguiente comando:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)